content/en/users/aai/check-in/vos/authorisation/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Authorisation using VO information
+linkTitle: Authorisation
+type: docs
+weight: 10
+aliases:
+  - /users/check-in/vos/authorisation
+description: >
+  Authorisation using VO information
+---
+
+**Authorisation** is the mechanism handling permissions and access control.
+While authentication deals with the identification of a user, authorisation
+determines and enforces what a user can do in the system.
+
+This is depicted in the figure below, where a user access an application and
+authenticates through Check-in. If the authentication is successful, Check-in
+will share agreed user information with the application. As part of this
+information are **the entitlements** of the user, which encode the group and
+role membership of the user.
+
+![Check-in flow](checkin_flow.jpg)
+
+Services and applications typically use membership information to **authorise
+user access** to protected resources, so the entitlements play a relevant role
+in the authorisation task.
+
+## Entitlements
+
+An entitlement is simply a string that encodes some user membership.
+Entitlements are formated as **URN**s, in this format:
+
+`<NAMESPACE>:group:<VO>[:<SUBGROUP>*][:role=<ROLE>]#<GROUP-AUTHORITY>`
+
+This format is described in the
+[AARC-G069](https://aarc-community.org/guidelines/aarc-g069/) specification:
+
+![Structure of an entitlement](entitlements.jpg)
+
+Using this information, an application can determine that the user belongs to
+the the group "cloud" of the VO "vo.example.eu" and has the "operator" role
+assigned. With this, the application can make an authorisation decision whether
```

Comment:
As general rule, new sentences should be started on new lines.
This makes the diff cleaner and helps us do reviews.